.github/workflows/CI.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
```

Comment:
Do we want to start running nightly on this?

Reply:
I intentionally turned that off as `continue-on-error` (known issue) doesn't work as expected would be kinda noisy.